using-keycloak/src/main/java/org/acme/keycloak/AdminResource.java

```diff
@@ -25,6 +27,7 @@
  * @author <a href="mailto:psilva@redhat.com">Pedro Igor</a>
  */
 @Path("/api/admin")
+@Authenticated
```

Comment:
This has not actually been wired up yet, it should be done before the next release though

Reply:
The question is: should it be there, or `@RoleAllowed` should be enough.

I believe `@RoleAllowed` should be enough. WDYT?

Reply:
@RolesAllowed("*") is the same